streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -633,6 +636,10 @@ public boolean isProcessable(final long wallClockTime) {
             return false;
         }
 
+        if (hasPendingTransaction) {
```

Comment:
Could we add a comment here to explain why a task is not processable under pending transaction?

Reply:
Sure. Just realizing that the variable name is not ideal -- it actually about an "pending TX _commit_"...